test/testAuthentication.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+require('assert');
+var chai = require('chai');
+var chaiHttp = require('chai-http');
+var start = require('../lib/app');
+var mainApp = start();
+chai.use(chaiHttp);
+if (!global.Promise) {
+  // Checks to see if there is a global promise
+  var q = require('q');
+  chai.request.addPromises(q.Promise);
+}
+var expect = chai.expect;
+
+describe('Test passport authentication', function () {
+
+  // Used for running crud need the values of what I created in the database
+  var server;
+  var chaiRequest;
+  var app;
+
+  before(function (done) {
+    server = mainApp.start(done);
+    app = mainApp.app;
+    chaiRequest = chai.request(app);
+  });
+
+  it('should render the not authorized page with a message saying you cannot log in until you have signed in or up', function (done) {
+    chaiRequest.get('/home')
+      .then(function (response) {
+        var notAuthorized = response.text.search('Not Authorized')
+        expect(notAuthorized).to.not.equal(-1); // would equal negative 1 if the word was not found
+        expect(response).to.have.status(200);
```

Comment:
other option here is to use `/Not Authorized/.test(response.text)` which will return true/false.